src/ia-book-actions.js

```diff
@@ -301,27 +307,24 @@ export default class IABookActions extends LitElement {
     this.loanRenewResult = this.loanRenewHelper.result;
   }
 
-  /**
-   * create / select the modal-manager component on DOM
-   * @memberof IABookActions
-   */
-  async useModalManager() {
-    if (!this.modal) {
-      this.modal = document.querySelector('modal-manager');
-
-      if (!this.modal) this.modal = document.createElement('modal-manager');
-    }
+  /** @returns HTMLElement */
+  get modal() {
+    const modalOnDom = document.querySelector('modal-manager');
```

Comment:
are we sure that everything that everytime there is modal-manager exist on /detailis page?

Reply:
yes, it will be required. i will add a check in JSIA.js to ensure.